server/routes/index.js

```diff
@@ -70,6 +76,32 @@ const mailchimp = async function(request, h) {
 const routes = {
   signup: require('./signup'),
   mailchimp,
+  'reCaptcha': async function(request, h) {
+    const payload = request.payload || {};
+    const reCaptchaToken = payload.reCaptchaToken || "";
```

Comment:
with Joi requiring this to be set, does it make sense to even have the `|| "";`

Reply:
Probably not. I guess if the joi stuff ever gets refactored, this'll fail a bit more gracefully?